test/common/generatePluginFixture.ts

```diff
@@ -0,0 +1,55 @@
+/** cspell:ignore promisify */
```

Comment:
Can we add some JSDoc for this file? I'm afraid it's not very clear what does it do.

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652